provider/v1alpha1/service.pb.go

```diff
@@ -0,0 +1,698 @@
+	file_provider_v1alpha1_service_proto_rawDesc = nil
+	file_provider_v1alpha1_service_proto_goTypes = nil
+	file_provider_v1alpha1_service_proto_depIdxs = nil
+}
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ context.Context
+var _ grpc.ClientConnInterface
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the grpc package it is being compiled against.
+const _ = grpc.SupportPackageIsVersion6
+
+// CSIDriverProviderClient is the client API for CSIDriverProvider service.
+//
+// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://godoc.org/google.golang.org/grpc#ClientConn.NewStream.
+type CSIDriverProviderClient interface {
+	// Version returns the runtime name and runtime version of the Secrets Store CSI Driver Provider
+	Version(ctx context.Context, in *VersionRequest, opts ...grpc.CallOption) (*VersionResponse, error)
+	// Execute mount operation in provider
+	Mount(ctx context.Context, in *MountRequest, opts ...grpc.CallOption) (*MountResponse, error)
+}
+
+type cSIDriverProviderClient struct {
+	cc grpc.ClientConnInterface
+}
+
+func NewCSIDriverProviderClient(cc grpc.ClientConnInterface) CSIDriverProviderClient {
+	return &cSIDriverProviderClient{cc}
+}
+
+func (c *cSIDriverProviderClient) Version(ctx context.Context, in *VersionRequest, opts ...grpc.CallOption) (*VersionResponse, error) {
+	out := new(VersionResponse)
+	err := c.cc.Invoke(ctx, "/v1alpha1.CSIDriverProvider/Version", in, out, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
+func (c *cSIDriverProviderClient) Mount(ctx context.Context, in *MountRequest, opts ...grpc.CallOption) (*MountResponse, error) {
+	out := new(MountResponse)
+	err := c.cc.Invoke(ctx, "/v1alpha1.CSIDriverProvider/Mount", in, out, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
+// CSIDriverProviderServer is the server API for CSIDriverProvider service.
+type CSIDriverProviderServer interface {
+	// Version returns the runtime name and runtime version of the Secrets Store CSI Driver Provider
+	Version(context.Context, *VersionRequest) (*VersionResponse, error)
+	// Execute mount operation in provider
+	Mount(context.Context, *MountRequest) (*MountResponse, error)
+}
+
+// UnimplementedCSIDriverProviderServer can be embedded to have forward compatible implementations.
+type UnimplementedCSIDriverProviderServer struct {
+}
+
+func (*UnimplementedCSIDriverProviderServer) Version(context.Context, *VersionRequest) (*VersionResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method Version not implemented")
+}
+func (*UnimplementedCSIDriverProviderServer) Mount(context.Context, *MountRequest) (*MountResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method Mount not implemented")
+}
+
+func RegisterCSIDriverProviderServer(s *grpc.Server, srv CSIDriverProviderServer) {
```

Comment:
Can we create a dummy server so that we can test the end to end and it can be used as a reference implementation of what the server should provide.